libpimeval/src/pimResMgr.h

```diff
@@ -71,6 +72,94 @@ class pimRegion
   bool m_isValid = false;
 };
 
+//! @class  pimDataHolder
+//! @brief  A container holding raw PIM data as bytes
+//! Assumption: Caller gurantees correct range and indices
+class pimDataHolder
+{
+public:
+  pimDataHolder(PimDataType dataType, uint64_t numElements)
+    : m_dataType(dataType),
+      m_numElements(numElements)
+  {
+    unsigned numBitsOfDataType = pimUtils::getNumBitsOfDataType(m_dataType);
+    m_bytesPerElement = (numBitsOfDataType + 7) / 8;
```

Comment:
Will you please add some comments on this calculation.

Reply:
Added more comments